packages/remote-server/src/swagger/x-methods.json

```diff
@@ -0,0 +1,202 @@
+{
+  "x-methods": {
+    "didManagerGetProviders": {
+      "description": "Returns a list of available identifier providers",
+      "arguments": {
+        "type": "object"
+      },
+      "returnType": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      }
+    },
+    "didManagerFind": {
+      "description": "Returns a list of managed identifiers",
+      "arguments": {
+        "$ref": "./components/schemas.json#/components/schemas/IDIDManagerFindArgs"
+      },
+      "returnType": {
+        "type": "array",
+        "items": {
+          "$ref": "./components/schemas.json#/components/schemas/IIdentifier"
+        }
+      }
+    },
+    "didManagerGet": {
+      "description": "Returns a specific identifier",
+      "arguments": {
+        "$ref": "./components/schemas.json#/components/schemas/IDIDManagerGetArgs"
+      },
+      "returnType": {
+        "$ref": "./components/schemas.json#/components/schemas/IIdentifier"
+      }
+    },
+    "didManagerCreate": {
+      "description": "Creates and returns a new identifier",
+      "arguments": {
+        "$ref": "./components/schemas.json#/components/schemas/IDIDManagerCreateArgs"
+      },
+      "returnType": {
+        "$ref": "./components/schemas.json#/components/schemas/IIdentifier"
+      }
+    },
+    "didManagerDelete": {
+      "description": "Deletes identifier",
+      "arguments": {
+        "$ref": "./components/schemas.json#/components/schemas/IDIDManagerDeleteArgs"
+      },
+      "returnType": {
+        "type": "boolean"
+      }
+    },
+    "dataStoreGetVerifiableCredential": {
+      "description": "Gets verifiable credential from the data store",
+      "arguments": {
+        "$ref": "./components/schemas.json#/components/schemas/IDataStoreGetVerifiableCredentialArgs"
+      },
+      "returnType": {
+        "$ref": "./components/schemas.json#/components/schemas/VerifiableCredential"
+      }
+    },
+    "dataStoreORMGetIdentifiersCount": {
+      "description": "Tries to obtain a count of ",
+      "arguments": {
+        "$ref": "./components/schemas.json#/components/schemas/FindIdentifiersArgs"
+      },
+      "returnType": {
+        "type": "number"
+      }
+    },
+    "dataStoreORMGetVerifiableCredentials": {
+      "description": "Returns a list of ",
+      "arguments": {
+        "$ref": "./components/schemas.json#/components/schemas/FindCredentialsArgs"
+      },
+      "returnType": {
+        "type": "array",
+        "items": {
+          "$ref": "./components/schemas.json#/components/schemas/UniqueVerifiableCredential"
+        }
+      }
+    },
+    "dataStoreORMGetVerifiableCredentialsCount": {
+      "description": "Returns a count of ",
+      "arguments": {
+        "$ref": "./components/schemas.json#/components/schemas/FindCredentialsArgs"
+      },
+      "returnType": {
+        "type": "number"
+      }
+    },
```

Comment:
I think some API descriptions are incomplete. Could you provide additional details to make them easier to understand?